crates/node/src/lib.rs

```diff
@@ -130,7 +131,7 @@ pub async fn start(config: NodeConfig) -> EyreResult<()> {
 
     #[expect(trivial_casts, reason = "Necessary here")]
     let mut server = Box::pin(calimero_server::start(
-        config.server,
+        server_config,
```

Comment:
this isn't defined anywhere?

did you mean?

```suggestion
        config.server.clone(),
```